src/FlatButton/FlatButton.js

```diff
@@ -93,6 +93,15 @@ class FlatButton extends Component {
      * Override the inline-styles of the root element.
      */
     style: PropTypes.object,
+    /**
+     * Target attribute applied to the button.
+     */
+    target: PropTypes.oneOf([
```

Comment:
I'm wondering if `string` isn't enough. I mean, do we have all the possible values? 
Looks like this list is far too restrictive https://www.w3.org/TR/html-markup/datatypes.html#common.data.browsing-context-name-or-keyword.